src/qoi/src/qoi_output.C

```diff
@@ -71,7 +71,7 @@ namespace GRINS
         if( comm.rank() == 0 )
           {
             std::ofstream output;
-            output.open( _file_prefix+".dat" );
+            output.open( _file_prefix+".dat",std::ofstream::app );
```

Comment:
Why force append?

Reply:
I guess I can add an addition `bool append = false` parameter so it can be toggled.

Reply:
Hmm. I guess it's not unreasonable to just append QoI values after each AMR step. And if you're not doing AMR, it won't matter. OK, I'm fine with just having it append.

Reply:
The only potential problem I can see is that if the file already exists from a previous run and we append to it, it might get confusing.

Reply:
Yeah, I thought about that, but if we're not appending, we're overwriting. Six in one, half dozen the other IMO, so instead of adding a bunch of extra code to worry about the case, let's just carefully document in the `master_example_inputfile.in`.

Reply:
Sounds good